zingolib/src/lib.rs

```diff
@@ -21,6 +21,48 @@ use std::{
 use tokio::runtime::Runtime;
 use zingoconfig::{ChainType, ZingoConfig};
 
+#[cfg(feature = "async_client")]
+pub async fn create_zingoconfdir_async(
+    server: http::Uri,
```

Comment:
This is the same code as the sync version, expect without the runtime. Can we dry this? Make them both call a shared helper, or remove the feature flag so that `create_zingoconf_with_datadir` can directly call this function?

Reply:
DRYed.